ai/src/generate_udiffs.py

```diff
@@ -0,0 +1,37 @@
+"""
+Generate udiffs for golden examples.
+
+I don't recommend running this script all the time as some of the udiffs
+for the goldens are generated by hand, in particular for:
```

Comment:
What's the problem with running this against udiffs that were generated by hand? What would happen?

Reply:
It will override the udiff generated by hand. In particular for `Move_the_icon_buttons_below_the_bot_text_Muy5HQ_20240901000` the udiff would not apply cleanly (i.e. computing the full output based on the udiff) and it's also not a very ideal udiff for LLM. output where it's fragmented.

Reply:
Longer-term, it'll be good to customize how we generate the udiffs, but looking at Python's built-in difflib it doesn't look trivial to customize the udiff logic so that it generates larger blocks of diffs that are more coherent for models to output (with the trade-off of having larger diff sizes)